### PR TITLE
Make multiple SCMs optional

### PR DESCRIPTION
**Pull request by reikoshea**
With the multiple-scms plugin being deprecated and this bug ( https://issues.jenkins-ci.org/browse/JENKINS-39598 ) causing /configure to fail to load, this PR will make the dependency optional.

### CODE REVIEW

**Comment by fznx-io** [comment]
Hi,
Any maintainer can approve this one please ? I think lot of people need this optional option for multiple scms plugin ..

**Review by fznx-io — APPROVED** [review]

**Review by Bufallo — APPROVED** [review]

**Comment by Joeb454** [comment]
Hi,

Is there any update on this or plans for this to make it into a release. As noted above, with the multiple SCMs plugin being marked as deprecated, it would be nice to be able to remove this for those of us that don't require it.

**Comment by unickq** [comment]
agree 👍 

@amansilla can you please check it?

**Comment by Ezhvsalate** [comment]
Any updates? Would like to remove multiple SCMs plugin too. Please review PR.

**Review by wrvdklooster — APPROVED** [review]

**Comment by timja** [comment]
@amansilla any chance of a review and a release?

File: src/main/java/org/jenkinsci/plugins/bitbucket/BitbucketBuildStatusHelper.java
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/bitbucket/BitbucketBuildStatusHelper.java:85]: won't this cause a class not found exception because of the import?
have you tested this without the plugin installed?

I would just remove the adaptor, from what I can tell the multi scm plugin is quite broken

**Comment by tedandrew** [comment]
Hello, any update on this? Would like to get rid of this deprecation warning already...
image

Starting to look into other options like https://github.com/030/bcbsn to replace this plugin, because of this issue.

P.S. Was looking through other closed PR to see if there is any other person that can review/approve PRs.
Looks like @amansilla you are the only reviewer here. Sorry to bother you, yet maybe you can help with reviewing this PR?

**Comment by derekatkins** [comment]
Looks like this should have been closed by https://github.com/jenkinsci/bitbucket-build-status-notifier-plugin/commit/8d78a0dafa02eef34c50b5f134e18107284380e2 
Still waiting on a new release, though...

**Comment by mkubenka** [comment]
As temporary solution it's possible to use development version from CI:

https://ci.jenkins.io/job/plugins/job/bitbucket-build-status-notifier-plugin/job/master/16/artifact/target/bitbucket-build-status-notifier.hpi

Upload file under `Dashboard` > `Plugin Manager` > `Advanced`

**Comment by reikoshea** [comment]
Closing PR. As noted this has been resolved by https://github.com/jenkinsci/bitbucket-build-status-notifier-plugin/commit/8d78a0dafa02eef34c50b5f134e18107284380e2

**Comment by derekatkins** [comment]
@reikoshea any chance for a new release?

**Comment by reikoshea** [comment]
> @reikoshea any chance for a new release?

I don't have control over that. To my knowledge, only @amansilla can cut releases. Since it's been 6 months since he's resolved this and still hasn't cut a release, I can only imagine we're not getting one.

**Comment by derekatkins** [comment]
@reikoshea ah well  For now I'm using the development version as suggested by @mkubenka above and it's been working well for me.

**Comment by amansilla** [comment]
I've tried a couple of times to release a new version but always getting errors during the process. I'm not the more experienced with the release process. Maybe someone can help me with that? Perhaps it would be a better option instead to have to release manually new releases, just migrate the manual process to GitHub actions? Any help will be welcome.

**Comment by timja** [comment]
Docs are here if either want to help out:
https://www.jenkins.io/doc/developer/publishing/releasing/

The releasing via GitHub actions is the easiest and most reliable: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

**Pull request closed without merging**
